site/faq.md

```diff
@@ -112,3 +112,30 @@ setting. Flux will read this value and parse the Kubernetes secret.
 
 For a guide showing how to do this, see the
 [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+### Can I temporarily make flux ignore a deployment?
+
+Yes. The easiest way to do that is to use the following annotation:
+
+```yaml
+    flux.weave.works/ignore: true
+```
+
+A live example of this can be seen
+[here](https://github.com/stefanprodan/openfaas-flux/blob/master/secrets/openfaas-token.yaml).
+
+Please note that [flux#749](https://github.com/weaveworks/flux/issues/749)
+limits this currently to resources of the type `namespace`, `service`
+and `deployment`.
```

Comment:
This isn't true; see #749, which is referenced in the above.

Reply:
Yes, I saw that. I was waiting for confirmation of @squaremo on https://github.com/weaveworks/flux/pull/1210#pullrequestreview-135073547